routes/routes.py

```diff
@@ -18,25 +18,41 @@ def __init__(self, db, executor, caching_manager):
 
     def register_routes(self):
         @api_bp.route('/train')
-        def train(): return self.training_service.train()
+        def train():
+            logging.info("The route '/train' has been called.")
+            return self.training_service.train()
 
         @api_bp.route('/find_recipe_in_text/', methods=['POST'])
-        def check_if_text_is_recipe(): return self.text_extractor_service.check_if_text_in_recipe(request.form)
+        def check_if_text_is_recipe():
+            logging.info("Route '/find_recipe_in_text/' has been called.")
+            return self.text_extractor_service.check_if_text_in_recipe(request.form)
```

Comment:
I have a small feeling this is not related to the PR as well.
remember you can always stash/ commit to another branch/ not commit irrelevant changes.
PR has to be focused on the task :)